packages/account/src/Components/poi/status/unsupported/card-details/selfie-upload.jsx

```diff
@@ -20,6 +39,9 @@ const SelfieUpload = ({ initial_values, goBack, onConfirm, onFileDrop }) => {
                 initialValues={initial_values || setInitialValues([SELFIE_DOCUMENT])}
                 validate={values => validateFields(values, undefined, [SELFIE_DOCUMENT])}
                 onSubmit={onConfirm}
+                innerRef={formik_actions =>
+                    formik_actions ? setFormikValues(formik_actions.values) : setFormikValues({})
+                }
```

Comment:
```suggestion
                innerRef={formik_actions => setFormikValues(formik_actions?.values || {})}
```

Reply:
Also, you need to do this setState conditionally because if it's not `is_pa_signup` then you are creating unnecessary rendering.